src/Core/Secrets/IConnectedModeSecrets.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.Core.Secrets
+{
+    public interface IConnectedModeSecrets
+    {
+        /// <summary>
+        /// Returns true if the project is bound to either SonarCloud, or to SonarQube version >= 9.9
+        /// Returns false if project is unbound or bound to SonarQube version <= 9.8
+        /// </summary>
+        bool AreSecretsAvailable();
```

Comment:
The summary is a bit too detailed - I would probably have made it simpler e.g.

```
<summary>Returns whether Secrets are supported in Connected Mode</summary>
```

... and moved your existing comment to `<remarks>` section.